src/common/routes/OnboardingRoutes.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
+import { RouteComponentProps } from 'react-router'
+import { Route, Switch, useLocation, useHistory } from 'react-router-dom'
+import { CSSTransition, TransitionGroup } from 'react-transition-group'
+
+import { pageRoutes } from './index'
+import * as Styles from './OnboardingRoutes.styles'
+import * as ROUTES from '../utils/constants/routes'
+
+type RouteType = {
+  id: string
+  path: string
+  component:
+    | React.FunctionComponent<RouteComponentProps>
+    | React.ComponentClass<RouteComponentProps>
+    | null
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    | any // TODO remove this any and eslint disable after user login is taken by redux
```

Comment:
Please remove `any`

Reply:
ok, but as in the comment, can we remove it when we implement the action?

> | any // TODO remove this any and eslint disable after user login is taken by redux